mozilla_vpn/views/test1.view.lkml

```diff
@@ -0,0 +1,274 @@
+view: test1 {
```

Comment:
I'm guessing this will just be temporary, otherwise this view should probably have a more descriptive name

Reply:
@scholtzan  I removed test1 view and explore files now and pushed.

Reply:
oh, you'll need to open another PR